### PR TITLE
cephfs: add examples for volumegroupsnapshot

### PR DESCRIPTION
**Pull request by yati1998**
this commit add yaml exampl files for creation of
volumegroupsnapshotclass and volumegroupsnapshot for cephfs driver

### CODE REVIEW

File: examples/cephfs/groupsnapshotclass.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: groupsnapshot.storage.k8s.io/v1alpha1
+kind: VolumeGroupSnapshotClass
+metadata:
+  name: csi-cephfsplugin-groupsnapclass
+driver: rook-ceph.cephfs.csi.ceph.com # csi-provisioner-name
```
Comment (Madhu-1) [inline examples/cephfs/groupsnapshotclass.yaml]: use the cephcsi driver name (not specific to Rook)

File: examples/cephfs/groupsnapshotclass.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+parameters:
+  # Specify a string that identifies your cluster. Ceph CSI supports any
+  # unique string. When Ceph CSI is deployed by Rook use the Rook namespace,
+  # for example "rook-ceph".
```
Comment (Madhu-1) [inline examples/cephfs/groupsnapshotclass.yaml]: Please remove this one  or update as per cephcsi doc clusterID

File: examples/cephfs/groupsnapshotclass.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  clusterID: rook-ceph # namespace:cluster
+  fsName: myfs
+  csi.storage.k8s.io/group-snapshotter-secret-name: rook-csi-cephfs-provisioner
+  csi.storage.k8s.io/group-snapshotter-secret-namespace: rook-ceph # namespace:cluster
```
Comment (Madhu-1) [inline examples/cephfs/groupsnapshotclass.yaml]: remove rook from the secret and remove comments related to Rook

**Review by Madhu-1 — CHANGES_REQUESTED** [review]

**Comment by nixpanic** [comment]
Looks good to me. Just make sure to address the YAML linter issue too

https://github.com/ceph/ceph-csi/actions/runs/10614805727/job/29421527358?pr=4814#step:3:1138

**Review by nixpanic — APPROVED** [review]

**Comment by yati1998** [comment]
@Madhu-1 @Rakshith-R can you please review the PR

**Review by Madhu-1 — APPROVED** [review]

**Comment by Madhu-1** [comment]
@mergifyio queue

**Pull request merged**